arctic/chunkstore/chunkstore.py

```diff
@@ -239,37 +242,43 @@ def read(self, symbol, chunk_range=None, filter_data=True, **kwargs):
         -------
         DataFrame or Series
         """
-
         sym = self._get_symbol_info(symbol)
         if not sym:
             raise NoDataFoundException('No data found for %s' % (symbol))
 
-        spec = {SYMBOL: symbol,
-               }
+        # special handling based on a single symbol or multi symbol read
```

Comment:
I wonder if it would cleaner to treat all cases as a list - e.g. place a single symbol in a one element list and then have a special case for the return value.